app/reflexes/admin/orders_reflex.rb

```diff
@@ -37,7 +37,8 @@ def bulk_invoice(params)
       BulkInvoiceJob.perform_later(
         params[:bulk_ids],
         "tmp/invoices/#{Time.zone.now.to_i}-#{SecureRandom.hex(2)}.pdf",
-        channel: SessionChannel.for_request(request)
+        channel: SessionChannel.for_request(request),
+        current_user_id: current_user.id
```

Comment:
You can pass a user object as well and Rails will get it from the database for you.

Reply:
That sounds really handy, I guess it would save us from having to define `BulkInvoiceJob#current_user`.